code/global/Utils.cs

```diff
@@ -157,5 +158,43 @@ public static float SourceUnitsToMeters(float sourceUnits)
         {
             return sourceUnits / 39.37f;
         }
+
+        public static T GetHoveringPanel<T>(Panel excludePanel, Panel rootPanel = null) where T : Panel
```

Comment:
Would it be worth making this param a List instead? So we can provide a list of panels to exclude?

Reply:
If needed, we can do. But currently, even not this function is in use. By default, Panel's Mouse events should be used (but these wouldn't work for panels, that's why I created it - then I came up with a better way, that is not as extensible tho).